### PR TITLE
blog: whats new may

### PR DESCRIPTION
**Pull request by alvarosabu**

### CODE REVIEW

**Comment by JaimeTorrealba** [comment]
Ready when you are @alvarosabu

File: src/content/blog/whats-new-may-2024.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+While in the core the V4 is cooking, in Cientos we aren't resting and so we have exiting news for you.
+
+## Features
+
+### Local canvas option for mouseparallax component
+
+Now we can select the [MouseParallax](https://cientos.tresjs.org/guide/abstractions/mouse-parallax.html) effect so that it is applied for a specific canvas or for the whole viewport.
+
+Made by [emavitta-remago](https://github.com/Tresjs/cientos/commits?author=andretchen0)
```
Comment (garrlker) [inline src/content/blog/whats-new-may-2024.md:21]: Just noticed emavitta was linking to Andretchen, it should be this

```suggestion
Made by [emavitta-remago](https://github.com/Tresjs/cientos/commits?author=emavitta-remago)
```

**Review by garrlker — CHANGES_REQUESTED** [review]
Everything else looks good besides that contributor link

**Comment by JaimeTorrealba** [comment]
Hi @garrlker thanks for point it, And sorry I left this one here open. This should be closed we decided to not do the blog post, instead a just tweeter/discord announce is enough for this (we will do one for the v4 of the core and another for v4 of cientos)

**Pull request closed without merging**